commons/src/main/java/fr/free/nrw/commons/category/AsyncResponse.java

```diff
@@ -0,0 +1,4 @@
+package fr.free.nrw.commons.category;
+
+public interface AsyncResponse {
```

Comment:
Ah, whoops, nope. That was a relic of my last draft, I had no idea this part was around before the commit that I branched my fresh start from. Deleted.